portal-frontend/src/app/features/home/application-list/application-list.component.ts

```diff
@@ -33,6 +33,8 @@ export class ApplicationListComponent implements OnInit {
     [APPLICATION_STATUS.INCOMPLETE, 'incomplete'],
     [APPLICATION_STATUS.WRONG_GOV, 'wrong-government'],
     [APPLICATION_STATUS.CANCELLED, 'cancelled'],
+    [APPLICATION_STATUS.ALCDECISION, 'submitted-to-alc'],
+    [APPLICATION_STATUS.CEODECISION, 'submitted-to-alc'],
```

Comment:
I assume the TODO is to add the statuses later?

Reply:
good catch, I forgot to add custom colour for portal